shared/src/main/java/gov/hhs/cdc/trustedintermediary/metadata/MetaDataStep.java

```diff
@@ -0,0 +1,4 @@
+package gov.hhs.cdc.trustedintermediary.metadata;
+
+/** List of steps where we log metadata events */
```

Comment:
Let's update this JavaDocs to talk about what this empty interface is for.

Reply:
done.